content/en/docs/release-notes/release-notes-0.16.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Release Notes"
+linkTitle: "v0.16"
+weight: 840
+type: "docs"
+---
+
+The `v0.16` release has a few focus areas:
+
+* Enable the new certificate controller for all users
+* `kubectl cert-manager create certificaterequest` for signing local certificates
+* `v1beta1` API
+
+
+As usual, please read the [upgrade notes](/docs/installation/upgrading/upgrading-0.15-0.16/) before upgrading.
+
+## New certificate controller
+
+The Certificate controller is one of the most commonly used controllers in the project.
+It represents the 'full lifecycle' of an x509 private key and certificate, including
+private key management and renewal.
+
+In `v0.15` we added these under a feature gate to allow users to test these and gather feedback.
+Thanks to everyone testing these and reporting issues we were able to fix issues and improve the controller.
+In `v0.16` this controller is now the default one in cert-manager. 
+
+For more information on this, we invite you to read our [design document](https://github.com/jetstack/cert-manager/pull/2753).
+
+
+## kubectl cert-manager tool for signing certificates
+
+cert-manager `v0.15` included a kubectl plugin that allowed to interact with cert-manager.
+In this release we leverage this plugin to allow users to sign certificates on their computer
+or inside the container itself.
+
+The `kubectl cert-manager create certificaterequest` command creates a new CertificateRequest 
+resource based on the YAML manifest of a Certificate resource as specified by `--from-certificate-file` flag.
+ 
+For example this will create a CertificateRequest resource with the name "my-cr" based on the Certificate described in `my-certificate.yaml` while storing the
+private key and x509 certificate in `my-cr.key` and `my-cr.crt` respectively.
+```console
+$ kubectl cert-manager create certificaterequest my-cr --from-certificate-file my-certificate.yaml --fetch-certificate --timeout 20m
+```
+
+The kubectl cert-manager binary can be downloaded from the [GitHub release page](https://github.com/jetstack/cert-manager/releases/)
```

Comment:
Placeholder: In case the plugin is added to krew-index before v0.16 release, do we want to mention it here?

Reply:
gonna replace it with `More information can be found on our [kubectl plugin page](../../usage/kubectl-plugin/).`